src/applications/personalization/connected-accounts/components/ConnectedApp.jsx

```diff
@@ -79,15 +79,15 @@ class ConnectedApp extends React.Component {
           <div className={`${cssPrefix}-row-details-block`}>
             <div className={`${cssPrefix}-row-details-block-content`}>
               <p>
-                <strong>{title}</strong>
-                &nbsp;can view your:
                 <button
                   aria-label={`Disconnect ${title} from your account`}
                   className="usa-button-primary"
                   onClick={this.openModal}
                 >
                   Disconnect
                 </button>
+                <strong>{title}</strong>
+                &nbsp;can view your:
```

Comment:
I changed the order of the disconnect button here because on small screens it was showing up in between the name of the app and the list of grants, which is pretty confusing. Let me know if that should change.

Before:
<img width="201" alt="Screen Shot 2019-03-12 at 6 52 26 PM" src="https://user-images.githubusercontent.com/473542/54243843-0f40e480-44f8-11e9-8e1b-5ea52124612d.png">

After:
<img width="200" alt="Screen Shot 2019-03-12 at 6 47 50 PM" src="https://user-images.githubusercontent.com/473542/54243851-17008900-44f8-11e9-8e69-12f50d84a092.png">